src/index.js

```diff
@@ -95,7 +98,7 @@ function isValidStaticPropertyName(name) {
 }
 
 function isValidRegex(pattern, flags) {
-  return true; // TODO fix this when pattern-acceptor is fixed
+  return acceptRegex(pattern, flags);
```

Comment:
Since this function just passes its arguments to another function, we can instead `import isValidRegex from "shift-regexp-acceptor";` and delete this.